apps/microtvm/zephyr/demo_runtime/boards/nucleo_f746zg.conf

```diff
@@ -15,16 +15,14 @@
 # specific language governing permissions and limitations
 # under the License.
 #
-# This file is specific to the nRF5340 DK board.
+# This file is specific to the STM32F746 Nucleo board.
 
 # For intrinsics used by generated optimized operators.
 CONFIG_CMSIS_DSP=y
 
-# Required for Cortex-M33 devices.
-CONFIG_MAIN_STACK_SIZE=50
-
 # For random number generation.
 CONFIG_ENTROPY_GENERATOR=y
+CONFIG_TEST_RANDOM_GENERATOR=y
```

Comment:
Not really, I'm using Zephyr 2.4.0, the HEAD is at:
```
commit 7a3b253ced7333f5c0269387a7f3ed1dee69739d (tag: zephyr-v2.4.0, tag: v2.4.0)
Author: Maureen Helm <maureen.helm@nxp.com>
Date:   Sat Sep 26 11:40:18 2020 -0500

    release: Zephyr 2.4.0
    
    Set version to 2.4.0.
    
    Signed-off-by: Maureen Helm <maureen.helm@nxp.com>
```
Maybe it's a difference regarding nucleo vs disco board on Zephyr. I know it's a bit unexpected because these boards are quite alike, but in the past I found some remarkable differences regarding the symbol addresses in `zephyr.elf`, for instance, so I would be not surprised if the kconfig files used are different and the one for disco board needs that config. On my last PR version I removed it for the nucleo boards and left it only for the disco boards.